DigitalLearningSolutions.Web/Controllers/TrackingSystem/Delegates/DelegateApprovalsController.cs

```diff
@@ -10,9 +12,22 @@
     [Route("/TrackingSystem/Delegates/Approve")]
     public class DelegateApprovalsController : Controller
     {
+        private readonly IDelegateApprovalsService delegateApprovalsService;
+
+        public DelegateApprovalsController(IDelegateApprovalsService delegateApprovalsService)
+        {
+            this.delegateApprovalsService = delegateApprovalsService;
+        }
+
         public IActionResult Index()
         {
-            var model = new DelegateApprovalsViewModel(new List<UnapprovedDelegate>());
+            var centreId = User.GetCentreId();
+
+            var delegates = delegateApprovalsService
+                .GetUnapprovedDelegatesWithCustomPromptAnswersForCentre(centreId)
+                .Select(d => new UnapprovedDelegate(d.Item1, d.Item2));

```

Comment:
I don't suppose there is a way to set up names for the returned list items just so that d.Item1 and d.Item2 can be renamed to something a little more obvious?

Reply:
I think I would have to replace the tuple with an fully specified class, which I didn't think was worth it for this but am open to being persuaded about.

Reply:
Fair point. If it gets reused elsewhere in the future, it might be worth it, but for now it's good as is.

Reply:
It's actually really easy to change the names, just change the interface on the DelegateApprovalService to:

public List<(DelegateUser delegateUser, List<CustomPromptWithAnswer> prompts)>
            GetUnapprovedDelegatesWithCustomPromptAnswersForCentre(int centreId);
(and potentially the method too)